src/main/java/dev/sigstore/fulcio/client/FulcioValidator.java

```diff
@@ -41,26 +36,13 @@ public class FulcioValidator {
   private final TrustAnchor fulcioRoot;
 
   public static FulcioValidator newFulcioValidator(
-      byte[] fulcioRoot, @Nullable byte[] ctfePublicKey)
+      byte[] fulcioRoot, byte @Nullable [] ctfePublicKey)
```

Comment:
this is real weird, spotless did this?

Reply:
nah that was IntelliJ. blows my mind too. but apparently you shouldn't put nullable in front of a primitive type